### PR TITLE
added 29 september blog

### PR DESCRIPTION
**Pull request by pythongiant**
added the blog post for the 29th September meetup.

### CODE REVIEW

File: content/2018-09-29-pydelhi-meetup.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Title: PyDelhi meetup at FUELED NOIDA
+Slug:  pydelhi-meetup-09-september-2018
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: It should be `pydelhi-meetup-09-september-2018` > `pydelhi-meetup-29-september-2018`

File: content/2018-09-29-pydelhi-meetup.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Author: Srihari Unnikrishnan
+Email: srihari.unnikrishnan@gmail.com
+Summary: PyDelhi Meetup on the 29th of september at fueled noida.
+
+![fueled](content/images/pydelhi-17-02-2018.jpg)
+
+
+So the machine learning themed pydelhi meetup kicked off with Vishal (https://github.com/vishalsangwan) giving a talk on OpenCV and facial recognition. 
+Vishal started off with first clearing some basics about video and image capturing where he talked about how a computer handles video input. He uses OpenCV to take in video input using his laptop camera. Afterwards he uses facial recognition datasets to track faces in his video input using his Rasberry pi and tried out different datasets.
+![talk-1]({filename}/images/first-t.jpg)
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: `first-t.jpg` is incorrect. This file has a `.png` extension. Also, name the file specific to the blog, for example, `29-09-2019-first-talk.png`.

File: content/2018-09-29-pydelhi-meetup.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Then there was a quick lightning talk given by Raju.
+
+Then we broke for snacks and networking involving a very intense foosball competition.
+
+The Second Talk was given by Praneet Nigam(https://github.com/Praneet460) following on the Machine Learning Train by giving a talk on Matplotlib, Seaborn and Pandas where he showcased some ways to data filtering and classification. Praneet Used a Pokemon dataset where he evaluated features of his pokemon dataset which included the height, weight and type of pokemeon. all required data for the talk can be found here.(https://github.com/Praneet460/DataVisualizationPyDelhi).
+
+We ended our machine learning themed meetup with community and hiring pitches.
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: `community` > `community mixer`

File: content/2018-09-29-pydelhi-meetup.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Then there was a quick lightning talk given by Raju.
+
+Then we broke for snacks and networking involving a very intense foosball competition.
+
+The Second Talk was given by Praneet Nigam(https://github.com/Praneet460) following on the Machine Learning Train by giving a talk on Matplotlib, Seaborn and Pandas where he showcased some ways to data filtering and classification. Praneet Used a Pokemon dataset where he evaluated features of his pokemon dataset which included the height, weight and type of pokemeon. all required data for the talk can be found here.(https://github.com/Praneet460/DataVisualizationPyDelhi).
+
+We ended our machine learning themed meetup with community and hiring pitches. 
+
+Until next time ,
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: `Until next time ,` > `Until next time,`

File: content/2018-09-29-pydelhi-meetup.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+Title: PyDelhi meetup at FUELED NOIDA
+Slug:  pydelhi-meetup-09-september-2018
+Date: 2018-09-29 11:00
+Category: General
+Author: Srihari Unnikrishnan
+Email: srihari.unnikrishnan@gmail.com
+Summary: PyDelhi Meetup on the 29th of september at fueled noida.
+
+![fueled](content/images/pydelhi-17-02-2018.jpg)
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: `content/images/pydelhi-17-02-2018.jpg` > `{filename}/images/pydelhi-17-02-2018.jpg`

File: content/2018-09-29-pydelhi-meetup.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+Title: PyDelhi meetup at FUELED NOIDA
+Slug:  pydelhi-meetup-09-september-2018
+Date: 2018-09-29 11:00
+Category: General
+Author: Srihari Unnikrishnan
+Email: srihari.unnikrishnan@gmail.com
+Summary: PyDelhi Meetup on the 29th of september at fueled noida.
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: Noun: a`fueled noida` > `Fueled Noida`

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please address these issues.

File: content/2018-09-29-pydelhi-meetup.md (hunk 1 above)
Comment (pythongiant) [reply content/2018-09-29-pydelhi-meetup.md]: oh thanks

File: content/2018-09-29-pydelhi-meetup.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+Title: PyDelhi meetup at FUELED NOIDA
+Slug:  pydelhi-meetup-29-september-2018
+Date: 2018-09-29 11:00
+Category: General
+Author: Srihari Unnikrishnan
+Email: srihari.unnikrishnan@gmail.com
+Summary: PyDelhi Meetup on the 29th of september at Fueled Noida.
+
+![fueled]({filename}/images/pydelhi-17-02-2018.png)
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: This is `.jpg`

File: content/2018-09-29-pydelhi-meetup.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+
+So the machine learning themed pydelhi meetup kicked off with Vishal (https://github.com/vishalsangwan) giving a talk on OpenCV and facial recognition. 
+Vishal started off with first clearing some basics about video and image capturing where he talked about how a computer handles video input. He uses OpenCV to take in video input using his laptop camera. Afterwards he uses facial recognition datasets to track faces in his video input using his Rasberry pi and tried out different datasets.
+![talk-1]({filename}/images/29-09-2019-first-talk.jpg)
```
Comment (realslimshanky) [inline content/2018-09-29-pydelhi-meetup.md]: This is `.png`

**Review by realslimshanky — CHANGES_REQUESTED** [review]

File: content/2018-09-29-pydelhi-meetup.md (hunk 8 above)
Comment (pythongiant) [reply content/2018-09-29-pydelhi-meetup.md]: sorry. my bad!

**Review by realslimshanky — APPROVED** [review]

**Comment by realslimshanky** [comment]
@pythongiant thank you for your contribution :taco:

**Pull request merged**